docs/content/filtering.md

```diff
@@ -750,7 +750,9 @@ Useful for debugging.
 
 The `--exclude-if-present` flag controls whether a directory is
 within the scope of an rclone command based on the presence of a
-named file within it.
+named file within it. The flag can be repeated to check for
+multiple file names, precense of any of them will exclude the
```

Comment:
Looks good though `precense` is spelled wrong it should be `presence` (it is usually you correcting my spelling mistakes but I did notice that one!)

Reply:
Thanks!